static/app/components/devtoolbar/components/releases/releasesPanel.tsx

```diff
@@ -45,36 +87,150 @@ function ReleaseHeader({release, orgSlug}: {orgSlug: string; release: Release})
         )}
       </ReleaseInfoHeader>
       <ReleaseInfoSubheader
-        style={{display: 'flex', flexDirection: 'column', alignItems: 'flex-start'}}
+        css={[resetFlexColumnCss, subtextCss, {alignItems: 'flex-start'}]}
       >
-        {release.versionInfo?.package && (
-          <PackageName>
-            <TextOverflow ellipsisDirection="left">
-              {release.versionInfo.package}
-            </TextOverflow>
-          </PackageName>
-        )}
-        <span style={{display: 'flex', flexDirection: 'row', gap: '3px'}}>
+        <span css={[resetFlexRowCss, {gap: 'var(--space25)'}]}>
           <TimeSince date={release.lastDeploy?.dateFinished || release.dateCreated} />
           {release.lastDeploy?.dateFinished &&
             ` \u007C ${release.lastDeploy.environment}`}
         </span>
       </ReleaseInfoSubheader>
-    </div>
+    </PanelItem>
+  );
+}
+
+function CrashFreeRate({
+  prevReleaseVersion,
+  currReleaseVersion,
+}: {
+  currReleaseVersion: string;
+  prevReleaseVersion: string | undefined;
+}) {
+  const {
+    data: currSessionData,
+    isLoading: isCurrLoading,
+    isError: isCurrError,
+  } = useReleaseSessions({
+    releaseVersion: currReleaseVersion,
+  });
+  const {
+    data: prevSessionData,
+    isLoading: isPrevLoading,
+    isError: isPrevError,
+  } = useReleaseSessions({
+    releaseVersion: prevReleaseVersion,
+  });
+
+  if (isCurrError || isPrevError) {
+    return null;
+  }
+
+  if (isCurrLoading || isPrevLoading) {
+    return (
+      <PanelItem css={{width: '100%', padding: 'var(--space150)'}}>
+        <Placeholder
+          height={placeholderHeight}
+          css={[
+            resetFlexColumnCss,
+            panelSectionCss,
+            panelInsetContentCss,
+            listItemPlaceholderWrapperCss,
+          ]}
+        />
+      </PanelItem>
+    );
+  }
+
+  const currCrashFreeRate = getCrashFreeRate(currSessionData);
+  const prevCrashFreeRate = getCrashFreeRate(prevSessionData);
+  const diff = (currCrashFreeRate - prevCrashFreeRate).toFixed(2);
```

Comment:
if the `.toFixed(2)` call here got moved down we could avoid the `parseFloat()` calls later on.

```
const sign = Math.sign(diff);

getDiff(toPercent(Math.abs(diff)), ... // this will go to 3 decimals though